server/src/main/java/org/opensearch/common/util/FeatureFlags.java

```diff
@@ -91,4 +96,6 @@ public static boolean isEnabled(String featureFlagName) {
     public static final Setting<Boolean> EXTENSIONS_SETTING = Setting.boolSetting(EXTENSIONS, false, Property.NodeScope);
 
     public static final Setting<Boolean> SEARCH_PIPELINE_SETTING = Setting.boolSetting(SEARCH_PIPELINE, false, Property.NodeScope);
+
+    public static final Setting<Boolean> IDENTITY_SETTING = Setting.boolSetting(IDENTITY, false, Property.NodeScope);
```

Comment:
I think we should remove this high level flag, what do you think?

My rational to remove, either an Identity plugin is installed or it isn't, when this feature is no longer experimental doesn't seem very clear.  If there are specific features I would rather have more traditional configuration on/off around them instead of this broader area.

Reply:
Yeah that sounds good to me.